src/site/content/en/blog/add-touch-to-your-site/index.md

```diff
@@ -0,0 +1,810 @@
+---
+title: Add touch to your site
+subhead: |
+  Touchscreens are available on more and more devices,  from phones up to desktop screens. Your app should respond to their touch  in intuitive and beautiful ways.
+date: 2014-01-01
+updated: 2018-09-20
+authors:
+  - mattgaunt
+description: |
+  Touchscreens are available on more and more devices,  from phones up to desktop screens. Your app should respond to their touch  in intuitive and beautiful ways.
+tags:
+  - blog
+---
+
+{% YouTube id="Rwc4fHUnGuU" %}
+
+
+Touchscreens are available on more and more devices, ranging from phones to
+desktop screens. When your users choose to interact with your UI, your app
+should respond to their touch in intuitive ways.
+
+## Respond to element states
+
+Have you ever touched or clicked an element on a web page and questioned
+whether the site actually detected it?
+
+Simply altering the color of an element as users touch or interact with parts
+of your UI gives a basic reassurance that your site is working. Not only does
+this alleviate frustration, it can also give a snappy and responsive feel.
+
+DOM elements can inherit any of the following states: default, focus, hover
+and active. To change our UI for each of these states, we need to apply styles
+to the following pseudo classes `:hover`, `:focus` and `:active` as shown below:
+
+```css
+.btn {
+  background-color: #4285f4;
+}
+
+.btn:hover {
+  background-color: #296CDB;
+}
+
+.btn:focus {
+  background-color: #0F52C1;
+
+  /* The outline parameter suppresses the border
+  color / outline when focused */
+  outline: 0;
+}
+
+.btn:active {
+  background-color: #0039A8;
+}
+
+```
+
+[Try it](https://googlesamples.github.io/web-fundamentals/fundamentals/design-and-ux/input/touch/states-example.html)
+
+<figure>
+{% Img src="image/T4FyVKpzu4WKF1kBNvXepbi08t52/BoIVrO4Nh0MdDMHofs6q.png", alt="Image illustrating different colors for button
+states", width="500", height="300" %}
+</figure>
+
+On most mobile browsers *hover* and/or *focus* states will apply to an element
+after it's been tapped.
+
+Consider carefully what styles you set and how they will look to the user after
+they finish their touch.
+
+Note: Anchor tags and buttons may have different behavior
+in different browsers, so assume in some cases **hover**
+will remain and in others **focus** will remain.
+
+### Suppressing default browser styles
+
+Once you add styles for the different states, you'll notice that most browsers
+implement their own styles in response to a user’s touch. This is largely
+because when mobile devices first launched, a number of sites didn’t
+have styling for the `:active` state. As a result, many browsers added
+additional highlight color or style to give the user feedback.
+
+Most browsers use the `outline` CSS property to display a ring around an
+element when an element is focused. You can suppress it with:
+
+```css
+.btn:focus {
+    outline: 0;
+
+    // Add replacement focus styling here (i.e. border)
```

Comment:
```suggestion
    /* Add replacement focus styling here (i.e. border) */
```

Reply:
CSS doesn't support single line comments starting with `//`, only block comments starting with `/*` and ending with `*/`.